### PR TITLE
[40] Calculate Product Shipping Cost

### PR DESCRIPTION
**Pull request by ainobs**
Screenshot (43)

Screenshot (45)
s://user-images.githubusercontent.com/64330690/93400206-2d1ea080-f877-11ea-98d6-7b05cbd21ab6.png)
Screenshot (44)

## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/victorAienobe/dom/product.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        button.disabled = false;
+        button.style.backgroundColor = 'goldenrod';
+        return true;
+    }
+}
+
+function shipping() {
+    if(price.innerHTML < 1000) {
+        shipCost.innerHTML = 100;
+    } else if(price.innerHTML >= 1000 && price.innerHTML < 20000) {
```
Comment (ToluAkin) [inline submissions/victorAienobe/dom/product.js:27]: `price.innerHTML <= 20000`

File: submissions/victorAienobe/dom/product.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        shipCost.innerHTML = 0.15 * price.innerHTML;
+    } else if(price.innerHTML >= 20000 && price.innerHTML < 100000) {
```
Comment (ToluAkin) [inline submissions/victorAienobe/dom/product.js:29]: `price.innerHTML <= 100000`

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/victorAienobe/dom/product.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        shipCost.innerHTML = 0.2 * price.innerHTML;
+    } else {
+        shipCost.innerHTML = 25000;
+    }
+    totalCost.innerHTML = Number(shipCost.innerHTML) + Number(price.innerHTML);
+}
+
+function validate(e) {
```
Comment (adaugochi) [inline submissions/victorAienobe/dom/product.js:37]: does this calculate the shipping and total cost in real-time?

Comment (ainobs) [reply submissions/victorAienobe/dom/product.js:37]: yes the calculation works in real time

File: submissions/victorAienobe/dom/product.js (hunk 1 above)
Comment (ainobs) [reply submissions/victorAienobe/dom/product.js]: thanks. I've made the corrections

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**